go.mod

```diff
@@ -84,7 +84,7 @@ require (
 	k8s.io/kops v0.0.0-20190222135932-278e6606534e
 	k8s.io/kube-openapi v0.0.0-20181018171734-e494cc581111 // indirect
 	k8s.io/kubelet v0.0.0-20190313123811-3556bcde9670
-	k8s.io/kubernetes v1.12.6 // indirect
+	k8s.io/kubernetes v1.12.6
```

Comment:
I don't understand, why was it indirect and not any more?

Reply:
Ok, so this is no longer indirect because we added `k8s.io/kubernetes/pkg/cloudprovider`.

Reply:
I don't think that's it. But you can ask `go mod` with `go mod why` (I am AFKbut I will do it later)